sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.New.cs

```diff
@@ -160,5 +160,86 @@ public void ActivityLinksIgnored(int maxLinksAllowed, string activitySourceName,
                 WriteEvent(18, maxLinksAllowed, activitySourceName, activityDisplayName);
             }
         }
+
+        [Event(19, Message = "Failed to parse redirect headers. Not user actionable.", Level = EventLevel.Informational)]
+        public void RedirectHeaderParseFailed()
+        {
+            if (IsEnabled(EventLevel.Informational))
```

Comment:
Why having this check?

Reply:
Thanks for catching this.
We're using the IsEnabled check to avoid doing unnecessary work.

Copy/paste,... I had this pattern in too many places.
This has been fixed :)